CHAID/CHAID.py

```diff
@@ -159,7 +170,13 @@ def from_pandas_df(df, i_variables, d_variable, alpha_merge=0.05, max_depth=2, m
         ind_df = df[i_variables]
         ind_values = ind_df.values
         dep_values = df[d_variable].values
-        return CHAID(ind_values, dep_values, alpha_merge, max_depth, min_sample, split_titles=list(ind_df.columns.values))
+        if independent_variable_types is not None:
+            independent_variable_types = [CHAIDVariableTypes[x] for x in independent_variable_types]
```

Comment:
Should this (`CHAIDVariableTypes[x]`) really be square brackets? Also piping in numbers seems a bit icky. Why not just make the developer either use the enum, or cast from string. Neither are neat but they are explicit.